### DISCUSSION TITLE
Spellchecker in Quarto when using R + VSCode

### DISCUSSION THREAD

**Discussion by werkstattcodes** [quarto-dev/quarto-cli#1218] [Q&A]
Hi,
Is there a way to enable a spellchecker when using Quarto in combination with R and VSCode? I installed the extension Code Spell Checker and added the extension qmd to the file formats in scope. But it seems - at least on my side - not to work in Quarto. Any recommendations? Many thanks.

**Comment by jjallaire** [comment]
I think it's just a matter of following the instructions for the Code Spell Checker extension. I got it to work:

<img width="611" alt="Screen Shot 2022-06-24 at 6 58 48 AM" src="https://user-images.githubusercontent.com/104391/175521572-430a18f8-7db9-445b-806f-81e5744e16f7.png">

There is an icon in the status bar you'll see when you have a qmd open. There are various paths from there but it is definitely possible to add `quarto` to the types of files to be spell checked.

**Reply by werkstattcodes** [reply]
many thanks. my bad & sorry for the hassle.

**Reply by Lightup1** [reply]
Hi @jjallaire , I wonder how to enable dictionary to specific file type. In my macine, for qmd file there only a few dictionaries are active.  I want to include Latex and julia dictionaries but don't know how to do that.
Do you have any suggestions on that?
